flink-libraries/flink-state-processing-api/src/main/java/org/apache/flink/state/table/SavepointConnectorOptions.java

```diff
@@ -26,7 +26,7 @@
 
 import static org.apache.flink.configuration.description.TextElement.code;
 
-/** Options for the state connector. */
+/** Options for the savepoint connector. */
```

Comment:
would state be better - if this cover checkpoints as well?

Reply:
This change is unrelated to the meta feature, just renamed everything to savepoint as we've agreed on. This is used by the connector...